weave-js/src/components/PagePanelComponents/Home/HomePreviewSidebar.tsx

```diff
@@ -53,6 +58,53 @@ const DashboardTemplateItem = styled(LayoutElements.HStack)`
 `;
 DashboardTemplateItem.displayName = 'S.DashboardTemplateItem';
 
+const DashboardTemplateItemText = styled(LayoutElements.Block)`
+  white-space: nowrap;
+  overflow: hidden;
+  text-overflow: ellipsis;
+  margin-bottom: 4px;
+`;
+DashboardTemplateItemText.displayName = 'S.DashboardTemplateItemText';
+
+const TabContentWrapper = styled.div`
+  overflow-y: scroll;
+  padding: 16px;
+  height: 100%;
+`;
+TabContentWrapper.displayName = 'S.TabContentWrapper';
+
+const Label = styled.span`
+  color: ${MOON_500};
+  font-size: 15px;
+  font-weight: 400;
+  line-height: 21px;
+`;
+Label.displayName = 'S.Label';
+
+const HomeExpressionPreviewPartsWrapper = styled.div`
+  height: 100%;
+
+  > .tw-style {
+    height: 100%;
+  }
+`;
+HomeExpressionPreviewPartsWrapper.displayName =
+  'S.HomeExpressionPreviewPartsWrapper';
+
+type Template = {
+  display_name: string;
+  description: string;
+  op_name: string;
+};
+
+const SEED_BOARD_OP_NAME = 'py_board-seed_board';
```

Comment:
This is fine for now, but it does feel like we could make this a bit more generic:
```
RECOMMENDED_BOARD_ORDER = ['py_board-open_ai_completions_monitor']
FALLBACK_BOARD_ID = 'py_board-seed_board'
```

Then the logic is essentially: recommend the first matching board in recommended_board_order, or the first non-fallback, else fallback. ... but this is fine - just calling out a way to generalize